src/tokens/ERC20RebaseDistributor.sol

```diff
@@ -116,17 +132,24 @@ abstract contract ERC20RebaseDistributor is ERC20 {
 
     function _exitRebase(address account) internal {
         uint256 rawBalance = ERC20.balanceOf(account);
-        uint256 shares = uint256(rebasingState[account].nShares);
+        RebasingState memory _rebasingState = rebasingState[account];
+        uint256 shares = uint256(_rebasingState.nShares);
         uint256 rebasedBalance = (shares * rebasingSharePrice) /
             START_REBASING_SHARE_PRICE;
+        if (rebasedBalance < rawBalance) {
+            rebasedBalance = rawBalance;
```

Comment:
make sure these lines are tested with codecov

Reply:
Added test cases to make sure all these are checked